tests/trust/test_trust.yml

```diff
@@ -36,10 +37,13 @@
 
     - name: add trust
       ipatrust:
+        ipaadmin_password: SomeADMINpassword
         realm: windows.local
         admin: Administrator
         password: secret_ad_pw
         state: present
+      register: result
```

Comment:
This is unexpected with the commit title.

Reply:
This is another change, which is not ready yet. Sorry for that.